sdks/java/extensions/google-cloud-platform-core/src/test/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtilTest.java

```diff
@@ -772,7 +773,11 @@ public void testGCSChannelCloseIdempotent() throws IOException {
         GoogleCloudStorageReadOptions.builder().setFastFailOnNotFound(false).build();
     SeekableByteChannel channel =
         new GoogleCloudStorageReadChannel(
-            null, "dummybucket", "dummyobject", null, new ClientRequestHelper<>(), readOptions);
+            null,
+            new StorageResourceId("dummybucket", "dummyobject"),
```

Comment:
can we maintain `testBucket` and `testObject` for consistency ?

Reply:
done!